scripts/release/deploycdn.js

```diff
@@ -25,8 +25,24 @@ const tasks = new Listr( [
 	{
 		title: 'Upload files to CDN.',
 		task: () => {
+			tools.shExec(
+				`aws s3 cp ./${ RELEASE_CDN_DIRECTORY }/ s3://${ CDN_S3_BUCKET }/ckeditor5/${ cdnVersion }/ ${ S3_COPY_ARGS } \
+				 --exclude "*" --include "*.js" --content-type 'text/javascript; charset=utf-8'`,
+				{ verbosity: 'error' }
+			);
```

Comment:
I think the redesigned flow misses the `*.d.ts` files.

![image](https://github.com/ckeditor/ckeditor5/assets/2270764/d55adce1-5126-417b-a94a-859afe6fc3f3)

Reply:
Also, the release instructions aim to be fully async. Hence, I propose a small update:

```suggestion
			await tools.shExec(
				`aws s3 cp ./${ RELEASE_CDN_DIRECTORY }/ s3://${ CDN_S3_BUCKET }/ckeditor5/${ cdnVersion }/ ${ S3_COPY_ARGS } \
				 --exclude "*" --include "*.js" --content-type 'text/javascript; charset=utf-8'`,
				{ verbosity: 'error', async: true }
			);
```